lib/debezium/decimal_test.go

```diff
@@ -84,10 +120,9 @@ func TestEncodeDecimal(t *testing.T) {
 	}
 
 	for _, testCase := range testCases {
-		encodedValue, err := EncodeDecimal(testCase.value, testCase.scale)
+		actual, err := encodeDecode(testCase.value, testCase.scale)
 		if testCase.expectedErr == "" {
-			decodedValue := DecodeDecimal(encodedValue, nil, int(testCase.scale))
-			assert.Equal(t, testCase.value, decodedValue.String(), testCase.name)
+			assert.Equal(t, testCase.value, actual, testCase.name)
```

Comment:
One thing I realized, we're not actually checking for the err

Reply:
Ooh yeah good call!